### PR TITLE
Examples: Support more video examples in Safari.

### LINKED ISSUE #15732: Video samples fail on iOS
##### Description of the problem

The video samples:

https://threejs.org/examples/webgl_materials_video.html

https://threejs.org/examples/webgl_video_panorama_equirectangular.html

no longer work on iOS (no idea if they ever did but I suspect they did)

##### Three.js version

- [x] Dev
- [x] r101
- [ ] ...

##### Browser

- [ ] All of them
- [x] Safari
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [x] iOS

### PR DESCRIPTION
**Pull request by Mugen87**
Finally fixed #15732

MP4s by @greggman

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev28/examples/webgl_video_panorama_equirectangular.html
https://raw.githack.com/Mugen87/three.js/dev28/examples/webgl_kinect.html
https://raw.githack.com/Mugen87/three.js/dev28/examples/webvr_video.html

File: examples/webgl_video_panorama_equirectangular.html [hunk 1]
```diff
@@ -62,14 +75,7 @@
...
-				var video = document.createElement( 'video' );
-				video.crossOrigin = 'anonymous';
-				video.width = 640;
-				video.height = 360;
-				video.loop = true;
-				video.muted = true;
-				video.src = 'textures/pano.webm';
-				video.setAttribute( 'webkit-playsinline', 'webkit-playsinline' );
+				var video = document.getElementById( 'video' );
 				video.play();
```
Comment (mrdoob) [inline examples/webgl_video_panorama_equirectangular.html]: You can remove this line by using the `autoplay` attribute in `<video>`.

Comment (Mugen87) [reply examples/webgl_video_panorama_equirectangular.html]: Um, with this change, the video does not play in Chrome and Safari anymore. 

Demo: https://jsfiddle.net/3f86dmxc/ (WebM only)

Comment (mrdoob) [reply examples/webgl_video_panorama_equirectangular.html]: Weird... Thanks for trying!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**